pkg/config/config.go

```diff
@@ -865,6 +865,10 @@ func InitConfig(config Config) {
 	config.SetKnown("process_config.internal_profiling.enabled")
 	config.SetKnown("process_config.remote_tagger")
 
+	// Process Discovery Check
+	config.BindEnvAndSetDefault("process_config.process_discovery.enabled", true)
```

Comment:
Discussed offline, let's change it to `false` for 7.32.

Reply:
Added in https://github.com/DataDog/datadog-agent/pull/9082/commits/9cdd6e15a23b74a03cd2d592893767693c8f1d03 and https://github.com/DataDog/datadog-agent/pull/9082/commits/4de258a84b04738925571b8fd18ea9202b751cfe